tools/wasme/pkg/defaults/defaults.go

```diff
@@ -21,3 +23,14 @@ func NewDefaultPuller() pull.ImagePuller {
 	res, _ := resolver.NewResolver("", "", true, false)
 	return pull.NewPuller(res)
 }
+
+var (
+	WasmeConfigDir       = home() + "/.wasme"
+	WasmeImageDir        = filepath.Join(WasmeConfigDir, "store")
+	WasmeCredentialsFile = filepath.Join(WasmeConfigDir, "credentials.json")
+)
```

Comment:
These values live in the `cli` module's defaults because that's the module that deals with `wasme` specifically. The `pkg` module is shared with other projects, for example [Gloo Edge](https://github.com/solo-io/gloo) which uses different defaults for these values when it imports this module. We don't want to import wasme's defaults into other projects, so we don't want these values here.

Reply:
Discussed offline - these values are already being pulled in as the defaults in the `pkg` module. While this behavior isn't ideal, this change doesn't actually alter the current behavior at all, just the dependencies, so it's ok. It is on the caller of `store.NewStore()` to default the value passed to the func before calling it, otherwise it will fall back to the wasme defaults.